src/werkzeug/useragents.py

```diff
@@ -84,8 +84,18 @@ def __call__(
                 break
         else:
             platform = None
+
+        # Except for Trident, all browser key words come after the last ')'
+        i = 0
```

Comment:
Let's make this name more obvious, something like `last_paren`.

Reply:
Sure